packages/web/components/place-limit-tool/index.tsx

```diff
@@ -51,7 +51,11 @@ export const PlaceLimitTool: FunctionComponent<PlaceLimitToolProps> = observer(
       }
```

Comment:
I can't comment on the useState hooks since they're not modified in this PR, but what if we initialize them as null then set them in the effects when the query loads? I'm assuming they will always need to be initialized from the result of the query. If not, please ignore this suggestion

Also, IMO it's safer to use chain denoms (coinMinimalDenoms) instead of token symbols as they're more reliable asset identifiers.

Reply:
> Also, IMO it's safer to use chain denoms (coinMinimalDenoms) instead of token symbols as they're more reliable asset identifiers.

I've been slowly transitioning this on other branches